oggm/tests/test_models.py

```diff
@@ -4308,6 +4308,9 @@ def test_hydro_out_from_no_glacier(self, hef_gdir, inversion_params, store_month
                            odf['liq_prcp_on_glacier'] +
                            odf['snowfall_off_glacier'] +
                            odf['snowfall_on_glacier'])
+        # this test fails because snowfall_on_glacier changes, as the
+        # formerly negative melt_on_glacier was added to snowfall_on_glacier
+        # is this a strict test??? (the remaining part of the test does not fail)
```

Comment:
Yes it's ok to remove the first time step - maybe just check that tot_prcp is constant after the first year

Reply:
ok, I only check now that tot_prcp is constant after the first year. This works, but I don't know if it always works. If `melt_on_glacier` is negative after the first year, then the tot_prcp wouldn't be constant anymore. But I guess in this idealised case, due to the constant climate, the `melt_on_glacier` also does not get negative?